guide/curve.rst

```diff
@@ -0,0 +1,64 @@
+Commitment Curve
+================
+
+Contributions on the A-Team, whether paid or volunteer, follow a participation and responsibility trajectory, or commitment curve, described by the stages below.  Each stage, with the exception of Explorer, is associated with a related badge which can be earned by performing all of the tasks listed for each stage.
+
+Explorer
+--------
+
+You're just beginning to investigate participation on the A-Team.
+
+Criteria:
+
+* Find bugs to work on
+* Setup IRC, bugzilla, hg/git, environment
+
+Associate
+---------
+
+You're beinning to contribute to the A-Team by getting involved with a project, working with some Mozillians, and resolving some bugs.
+
+Criteria:
+
+* Work with 2+ mozillians
+* Resolve 3+ good first bugs
+* Resolve at least 1 mentored bug which isn't a good first bug
+* Get involved in a project
+
+Core Contributor
+----------------
+
+You've reached the point where you're a solid and consistent contributor to the A-Team.  You regularly participate in team meetings; file, review, and resolve bugs; and expand the number of Mozillians you work with.
+
+Criteria:
+
+* Create a Mozillians profile
+* Resolve 5+ good next bugs
```

Comment:
that seems good!

On Mon, Jan 12, 2015 at 7:59 PM, William Lachance <notifications@github.com>
wrote:

> In guide/curve.rst
> <https://github.com/mozilla/ateam-bootcamp/pull/2#discussion-diff-22836564>
> :
>
> > +Criteria:
> > +
> > +* Work with 2+ mozillians
> > +* Resolve 3+ good first bugs
> > +* Resolve at least 1 mentored bug which isn't a good first bug
> > +* Get involved in a project
> > +
> > +Core Contributor
> > +----------------
> > +
> > +You've reached the point where you're a solid and consistent contributor to the A-Team.  You regularly participate in team meetings; file, review, and resolve bugs; and expand the number of Mozillians you work with.
> > +
> > +Criteria:
> > +
> > +* Create a Mozillians profile
> > +* Resolve 5+ good next bugs
>
> Can we change this to 5+ bugs (not good first bugs)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/ateam-bootcamp/pull/2/files#r22836564>.
>